src/Components/Components/test/Routing/RouteTableFactoryTests.cs

```diff
@@ -1029,8 +1029,10 @@ public void DoesNotThrowIfStableSortComparesRouteWithItself()
     [Theory]
     [InlineData("/literal", "/Literal/")]
     [InlineData("/{parameter}", "/{parameter}/")]
+    [InlineData("/{parameter}part", "/part{parameter}")]
```

Comment:
Here's an interesting case I wanted to document in tests. In this case the route is counted as duplicate even though it's debatable if parameter is non-empty.

To be clear all tests were run before and after and pass in both.

Reply:
It's ambiguous because if we sent "/partpart" will match both.